address-space-controller/src/main/java/io/enmasse/controller/common/KubernetesHelper.java

```diff
@@ -363,8 +363,9 @@ private boolean isRBACSupported() {
 
     private void createRoleBinding(String name, String namespace, String refKind, String refName, List<Subject> subjectList) {
 
-        String apiVersion = client.isAdaptable(OpenShiftClient.class) ? "v1" : "rbac.authorization.k8s.io/v1beta1";
-        String apiPath = client.isAdaptable(OpenShiftClient.class) ? "/oapi/v1" : "/apis/rbac.authorization.k8s.io/v1beta1";
+        Boolean isOpenShift = client.isAdaptable(OpenShiftClient.class);
+        String apiVersion = isOpenShift ? "v1" : "rbac.authorization.k8s.io/v1beta1";
+        String apiPath = isOpenShift ? "/oapi/v1" : "/apis/rbac.authorization.k8s.io/v1beta1";
```

Comment:
Why v1beta1 instead of v1? I think we should use a stable api available in 1.9.

Reply:
Ah, right. Will change to v1.